README.md

```diff
@@ -1,2 +1,77 @@
-# yoti-ruby-sdk-sandbox
-Yoti Ruby SDK Sandbox
+# Yoti Ruby Sandbox SDK
+
+This repository contains the tools you need to test your Yoti integration.
+
+## Installing the Sandbox
+
+To import the Yoti Sandbox SDK inside your project, add this line to your application's Gemfile:
+
+```ruby
+gem 'yoti_sandbox'
+```
+
+And then execute:
+
+```shell
+bundle install
+```
+
+Or simply run the following command from your terminal:
+
+```shell
+gem install yoti_sandbox
+```
+
+## Configuration
+
+* `CLIENT_SDK_ID` is the SDK identifier generated by Yoti Hub in the Key tab when you create your app.
```

Comment:
Would be good to distinguish between regular and Sandbox SDK IDs. I realise it's like this in other ones too, so if this sounds ok we can update the rest
```suggestion
* `CLIENT_SDK_ID` is the Sandbox SDK identifier generated by Yoti Hub from the Sandbox section. This is different to the Client SDK ID for your application.
```

Reply:
Could even update the var name to `SANDBOX_CLIENT_SDK_ID`? As I know people won't always read the READMEs. @jlmartyres what do you think?

Reply:
That's a good idea, and SANDBOX_CLIENT_SDK_ID is consistent with the Hub

Reply:
Updated in 5336112